### PR TITLE
Add Honeywood Mainnet

### PR DESCRIPTION
**Pull request by Melekhin**

### CODE REVIEW

File: cosmos/honeywood.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "chainId": "HoneyWood",
```
Comment (Melekhin) [inline cosmos/honeywood.json:2]: @HeesungB 
Hello, 
I saw that my pipelines are not passing and it is due to the fact that the chainId should be in lower case. But the chainID looks exactly like this 'HoneyWood' in CamelCase

**Comment by HeesungB** [comment]
@Melekhin Could you remove `coinGeckoId` field?
I think that It is invalid.

**Comment by HeesungB** [comment]
@Melekhin Could you check about image url?
I think the image folder name contains capitalization, so the URL should change as well.

**Comment by HeesungB** [comment]
@Melekhin Thank you for contribution!

**Pull request merged**